bundles/io/org.eclipse.smarthome.io.json/src/main/java/org/eclipse/smarthome/io/json/JsonBindingService.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.io.json;
+
+import java.io.Reader;
+import java.io.Writer;
+import java.lang.reflect.Type;
+
+/**
+ * JSON data binding layer.
+ *
+ * @author Flavio Costa - Initial contribution.
+ *
+ * @param <T> Type to be serialized or deserialized.
+ */
+public interface JsonBindingService<T> {
+
+    /**
+     * Serialize an object to Json.
+     *
+     * @param object Object instance to be serialized.
+     * @param genericType Generic type definition.
+     * @param writer Writer where the Json output will be written to.
+     * @return Array of bytes containing the serialized Json.
```

Comment:
The return type is void

Reply:
Thanks, `@return` is removed from the Javadoc